ros2_batch_job/__main__.py

```diff
@@ -45,14 +45,15 @@
 sys.stderr = UnbufferedIO(sys.stderr)
 
 pip_dependencies = [
+    'EmPy',
```

Comment:
should dependencies be listed in alphabetical order regardless of the case? (it seems that's how flake8 import order interprets alphabetical order according to the fixes in the attached PRs e.g. https://github.com/ros2/rosidl_dds/pull/40/files)